HEN_HOUSE/doc/src/pirs794-dosxyznrc/inputs/dosxyznrc.inp

```diff
@@ -1,12 +1,12 @@
 \begin{verbatim}
 *******************************************************************************
 
-            *********************************************
-            *                                           *
-            * dosxyznrc.mortran (Revision: 1.53 )       *
-            * (last edited Date: 2013/05/08 18:58:44 )  *
-            *                                           *
-            *********************************************
+            *********************************************      
+            *                                           *      
+            * dosxyznrc.mortran ($Revision: 1.56 $)     *      
+            * (last edited $Date: 2014/10/08 15:12:01 $)*      
```

Comment:
Why the edit date change to 2014?

Reply:
That was the last edit made under the old CVS system, when revision no. and date were tracked in the comments at the top of every file.  These lines have always been included with those describing the code inputs that are echoed into dosxyznrc.inp to be spat out in the manual (see older versions of this manual).  Evidently, it's time to delete these lines from dosxyznrc.mortran (and possibly some other codes)!

Reply:
Please remove the EOL white spaces.